server/helpers/utils.ts

```diff
@@ -11,6 +11,13 @@ function deleteFileAsync (path: string) {
     .catch(err => logger.error('Cannot delete the file %s asynchronously.', path, { err }))
 }
 
+function clearFile (filePath: string) {
```

Comment:
Doesn't seem to be used

Reply:
It is, its import is just renamed: https://github.com/Chocobozzz/PeerTube//blob/0379f0762d2002b191253f689153864b45baafa7/server/middlewares/validators/videos/videos.ts#L62

Reply:
Okay thanks, then why don't you use `remove()` from fs extr?

Reply:
I though unlink had a special meaning, but it seems we can use just use `deleteFileAsync` from the existing utils…